CK3toEU4/Data_Files/blankMod/output/common/cultures/!.expanded_culture_groups.txt

```diff
@@ -26,12 +26,6 @@ british = {
 		dynasty_names = { Aelfwig Alfstan Athelred Auchenceter Auchterworthy Bloccsham Brownlowe Burgwine Cerdic Coldren Cuthberht Cwmcott Ecgred Estmund Gardyner Glover Godwinsson Gorbold Hrothsige Hwitmann Lea Leofwine Purcell Unred Wolfsige Wulfstan Wyndham Wynstan Wyther }
 	}
 }
-gaelic = {
```

Comment:
No, you cannot remove the culture GROUP from here, read the text in the file header.